docs/get_started/get_started_demos.md

```diff
@@ -90,29 +96,26 @@ This guide uses the OpenVINO™ Model Downloader to get pre-trained models. You
 
    .. code-block:: sh
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 .. tab:: Windows
 
    .. code-block:: bat
 
-      cd <INSTALL_DIR>\deployment_tools\tools\model_downloader\
-      python info_dumper.py --print_all
+      info_dumper --print_all
 
 .. tab:: macOS
 
    .. code-block:: sh
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 @endsphinxdirective
 
-### Use `grep` to list models that have a specific name pattern
+### Use 'grep' to list models that have a specific name pattern
```

Comment:
```suggestion
### Use grep to List Models of a Same Naming Pattern
```

You can use the following command to list models of a same naming pattern:

Reply:
I think this heading and the heading above "List the models available in the downloader" should not be headings. They seem to be two options to find models. and the text after them is another topic which should have a heading. Please check.

Reply:
reworked